src/components/CreateMenu.js

```diff
@@ -27,6 +32,11 @@ const propTypes = {
 
     ...windowDimensionsPropTypes,
 };
+
+const defaultProps = {
+    popOverType: CONST.MODAL.MODAL_TYPE.POPOVER_LEFT_DOCKED,
+};
+
 class CreateMenu extends PureComponent {
```

Comment:
Is there a reason this is a `PureComponent` instead of just a normal `Component`?

Reply:
The pure component was implemented beforehand, not sure why it was done though.

Reply:
OK, I'll check on that and dig a little deeper. No need to let this block your PR since you didn't add it.